db/migration/1742925758569-ethereumFieldsToTable.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class EthereumFieldsToTable1742925758569 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            ALTER TABLE "discord_member"
+            ADD COLUMN "ethereumNetwork" character varying,
+            ADD COLUMN "ethereumWalletAddress" character varying
```

Comment:
We create a document everytime a user wants to connect on a network. So we don't need to create new fields for each network. You can simply write the data inside the "starknetNetwork" and "starknetWalletAddress" fields. It'll be easier for us to integrate several networks that way. (I'll make an issue for updating the naming because obviously we are going to store any kind of network, it's not just starknet anymore).